web-ui/src/main/resources/catalog/views/default/templates/results.html

```diff
@@ -72,7 +72,8 @@
 
   <button data-ng-click="toggleMap()"
           title="{{'toggleMiniMap' | translate}}"
-          class="gn-minimap-toggle btn btn-link">
-    <i class="fa fa-angle-double-right"></i>
+          class="gn-minimap-toggle btn btn-link minimap-button">
+    <i class="fa fa-angle-double-right" ></i>
+    <b class="gn-minimap-toggle minimap-text">MAP</b>
```

Comment:
Missing translate directive and translate key in JSON. If you really need uppercase, probably better to use css?